storybook/src/stories/Filters/molecules/DatePicker.stories.tsx

```diff
@@ -17,34 +18,42 @@ export default {
   }
 };
 
-const exampleCallback =  <T extends Function>(fn: T): T => {
+const exampleCallback = <T extends Function>(fn: T): T => {
   /** A toString to render the function in storybook */
   // eslint-disable-next-line no-param-reassign
   fn.toString = () => '   updateCallback: (data: DateInterval | Date) => {}';
   return fn;
 };
 
 export const _DatePicker = () => {
-
+  const language = select("Language", { English: 'english', Japanese: 'japanese' }, "japanese");
   const dateMode = select("Date Mode", { single: "single", interval: "interval" }, "interval");
   const timeMode = select("Time Mode", { off: "off", single: "single", interval: "interval" }, "interval");
   const dateTimeTextUpper = text('Date Time Text Upper', 'From');
   const dateTimeTextLower = text('Date Time Text Lower', 'To');
   const timeZoneTitle = text('Time Zone Title', 'Timezone');
-  const timeZoneValueTitle = text('Time Zone Value','JST');
-  const updateCallback = action('Date / Time Updated')
+  const timeZoneValueTitle = text('Time Zone Value', 'JST');
+  const updateCallback = action('Date / Time Updated');
+
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  const monthsList = object('Translate Months', jpMonths);
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  const daysList = object('Translate Days', jpDays);
```

Comment:
Why lint is disabled here? Please fix it.
// eslint-disable-next-line @typescript-eslint/no-unused-vars

Reply:
Done